libmamba/include/mamba/util/graph.hpp

```diff
@@ -539,6 +540,9 @@ namespace mamba::util
             const typename Graph::adjacency_list& adjacency
         )
         {
+            assert(status.size() == graph.successors().size());
+            assert(adjacency.size() == graph.successors().size());
+            assert(start <= status.size());
```

Comment:
Shouldn't it be a strict comparison here?

Reply:
Yes, thanks!